### DISCUSSION TITLE
Help Setting Replicas to 0

### DISCUSSION THREAD

**Discussion by JSylvia007** [pfelk/pfelk#414] [General]
Is there a way that I can set the replicas to 0 globally?  It seems as though my global template isn't working...  I was using this:

```
PUT _template/zeroreplicas
{
  "order": 1,
  "index_patterns": ["*"],
  "settings": {
    "index": {
      "number_of_replicas": "0"
    },
    "mappings": {},
    "aliases": {}
  }
}
```

**Comment by a3ilson** [comment]
@JSylvia007 - Is the template above linked to the pfelk index templates?

# Solution:

## Method 1 (easiest) 
- Amend the component template: pfelk-mappsing-ecs 
  - amend with `"number_of_replicas": "0"`

  ![Screen Shot 2022-02-21 at 8 10 02 PM](https://user-images.githubusercontent.com/16884679/155045198-14ce976f-ff58-4c6d-a315-460686887bd7.png)


## Method 2
- Create component template:
  - input the specified settings (e.g:
```
{
  "index": {
    "number_of_replicas": "0"
  }
}
```
### or
  - Navigate to 'Dev Tools' and input:
```
PUT _component_template/zeroreplicas
{
  "template": {
    "settings": {
      "index": {
        "number_of_replicas": "0"
      }
    }
  }
}
```
  - Then edit the desired index template and add the zeroreplicas component template
![Screen Shot 2022-02-21 at 8 07 49 PM](https://user-images.githubusercontent.com/16884679/155045063-944108d7-26ca-4b27-9359-d79aa91e6610.png)


- Added to [Wiki](https://github.com/pfelk/pfelk/wiki/How-To:-Set-Replicas-to-0)
